inginious/frontend/templates/course_admin/task_list.html

```diff
@@ -113,4 +118,48 @@ <h5 class="modal-title" id="taskDispenserModalLabel">{{_("Switch task dispenser"
         </button>
     </div>
 </div>
+
+<script>
+	const tasksWithErrors = document.getElementsByClassName("bg-danger")
```

Comment:
This whole piece of code is vanilla javascript. I'd recommend you to make it more jQuery-idiomatic to remain consistent with other parts of the code and ensure maximum compatibility with supported jQuery browsers/devices.